app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  belongs_to :user, :post
```

Comment:
- [ ] We can't write multiple belongs to in the same line. This syntax raises an error in the console
      Please separate them: 

      ```ruby
      belongs_to :user
      belongs_to :post
      ```